src/main/kotlin/com/chillibits/coronaaid/repository/HistoryRepository.kt

```diff
@@ -1,6 +1,13 @@
 package com.chillibits.coronaaid.repository
 
 import com.chillibits.coronaaid.model.db.HistoryItem
+import com.chillibits.coronaaid.model.dto.HistoryItemDto
 import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
 
-interface HistoryRepository: JpaRepository<HistoryItem, Int>
\ No newline at end of file
+interface HistoryRepository: JpaRepository<HistoryItem, Int> {
+
+    @Query("SELECT it FROM HistoryItem it, Infected i WHERE it.infectedId = i.id AND i.id = ?1")
```

Comment:
Does this also work with "SELECT it FROM HistoryItem it WHERE it.infectedId = ?1"?

Reply:
Most likely. I copied this code from the TestRepository, so we could change it there too.

Reply:
Yes

Reply:
Could you change that in both repositories before we merge this pr?

Reply:
Done